migrations/1584966605306-runIndexSync2.js

```diff
@@ -0,0 +1,34 @@
+require('mongoose');
+// eslint-disable-next-line no-unused-vars
+const { info, error } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+
+const {
+	homeworkModel,
+	submissionModel,
+	commentModel,
+} = require('../src/services/homework/model');
```

Comment:
Please don't do this. This will lead to "model already compiled" errors if you try to run this locally together with other migrations.

Reply:
Better: import the schema and build the model here, or build a fresh schema+model in the migration